modules/oauth/src/main/java/org/mule/extension/oauth2/internal/AbstractTokenRequestHandler.java

```diff
@@ -32,29 +34,81 @@
 import org.mule.runtime.api.tls.TlsContextFactory;
 import org.mule.runtime.core.api.DefaultMuleException;
 import org.mule.runtime.core.api.Event;
+import org.mule.runtime.core.api.Event.Builder;
 import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.context.MuleContextAware;
 import org.mule.runtime.core.api.message.InternalMessage;
+import org.mule.runtime.core.api.registry.RegistrationException;
+import org.mule.runtime.core.util.CollectionUtils;
+import org.mule.runtime.extension.api.annotation.Alias;
+import org.mule.runtime.extension.api.annotation.param.Optional;
+import org.mule.runtime.extension.api.annotation.param.Parameter;
 import org.mule.service.http.api.HttpService;
 import org.mule.service.http.api.client.HttpClient;
 import org.mule.service.http.api.client.HttpClientConfiguration;
 import org.mule.service.http.api.domain.message.response.HttpResponse;
 
 import java.io.IOException;
+import java.io.InputStream;
 import java.net.CookieManager;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.concurrent.TimeoutException;
+import java.util.function.Function;
 
-import javax.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public abstract class AbstractTokenRequestHandler implements Initialisable, Startable, Stoppable {
+public abstract class AbstractTokenRequestHandler implements Initialisable, Startable, Stoppable, MuleContextAware {
```

Comment:
this class is using ext-api annotation but it not in an extension but in a module.